build/flatpak/com.authormore.penpotdesktop.metainfo.xml

```diff
@@ -53,6 +53,32 @@
         <internet>offline-only</internet>
     </supports>
     <releases>
+        <release version="v0.7.0" date="2024-12-14">
```

Comment:
@Beryesa I suspect changelog has to be available on a tag, hence I added the notes.

Do we need anything else in this repo for Flathub release?

Reply:
The goal was to not need anything else in this repo, and I'd only kindly ask to add a changelog right before the tagged release, which you've done already, thanks!

Reply:
Thank you, that’s perfect.
Sure, I can do notes with a release preparation.